### PR TITLE
First stable release will be 1.0.0

### PR DESCRIPTION
**Pull request by chapulina**
Let's go full stable on Edifice.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-utils0 VERSION 0.1.0)
+project(ignition-utils VERSION 1.0.0)
```
Comment (scpeters) [inline CMakeLists.txt:6]: this also drops the major version from the project name. I think this is intentional, but I want to confirm

cc @j-rivero

Comment (chapulina) [reply CMakeLists.txt:6]: Good point, `ign-plugin` does have the 1. Adding it back

**Comment by chapulina** [comment]
* Windows CI including https://github.com/ignition-tooling/release-tools/pull/378 and https://github.com/ignition-tooling/gazebodistro/pull/22: [Build Status](https://build.osrfoundation.org/job/ign_utils-pr-win/19/)
* Ubuntu CI including https://github.com/ignition-tooling/release-tools/pull/378 [Build Status](https://build.osrfoundation.org/job/ignition_utils-ci-pr_any-ubuntu_auto-amd64/17/)

File: CMakeLists.txt (hunk 1 above)
Comment (azeey) [reply CMakeLists.txt]: So, to clarify, are we putting `1` in the debian package name as well? I see the homebrew package does have the version number.

**Comment by caguero** [comment]
> * Windows CI including [ignition-tooling/release-tools#378](https://github.com/ignition-tooling/release-tools/pull/378) and [ignition-tooling/gazebodistro#22](https://github.com/ignition-tooling/gazebodistro/pull/22): [Build Status](https://build.osrfoundation.org/job/ign_utils-pr-win/19/)
> * Ubuntu CI including [ignition-tooling/release-tools#378](https://github.com/ignition-tooling/release-tools/pull/378) [Build Status](https://build.osrfoundation.org/job/ignition_utils-ci-pr_any-ubuntu_auto-amd64/17/)

Is Homebrew supposed to be green or not ready yet?

File: CMakeLists.txt (hunk 1 above)
Comment (chapulina) [reply CMakeLists.txt]: I'm following `ign-plugin`'s pattern:

* homebrew: 1
* deb: no number
* cmake: 1

**Comment by chapulina** [comment]
> Is Homebrew supposed to be green or not ready yet?

It works with https://github.com/ignition-tooling/release-tools/pull/378 in: [Build Status](https://build.osrfoundation.org/job/ignition_utils-ci-pr_any-homebrew-amd64/19/)

**Review by azeey — APPROVED** [review]

**Review by j-rivero — APPROVED** [review]

**Comment by chapulina** [comment]
CI is :green_circle:

**Pull request merged**